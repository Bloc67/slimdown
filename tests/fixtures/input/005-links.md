A [link](http://www.google.com) and [another link](http://yahoo.com/path/_with_/underscores).