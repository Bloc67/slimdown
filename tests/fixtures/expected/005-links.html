<p>A <a href='http://www.google.com'>link</a> and <a href='http://yahoo.com/path/_with_/underscores'>another link</a>.</p>